### PR TITLE
Change subtext of  `dashboard creation modal`

### PR DESCRIPTION
**Pull request by marcel-bitfly**
When button is disabled show: "Coming soon" otherwise the `network name`

See: BIDS-3205

### CODE REVIEW

File: frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue [hunk 1]
```diff
@@ -3,14 +3,16 @@ import { IconNetwork } from '#components'
...
+
 const { currentNetwork, isMainNet } = useNetworkStore()
 
 // TODO: get the list from the API...
 let ValidatorDashboardNetworkList: ChainIDs[]
 if (isMainNet()) {
   ValidatorDashboardNetworkList = [ChainIDs.Ethereum, ChainIDs.Gnosis]
 } else {
-  ValidatorDashboardNetworkList = [ChainIDs.Holesky, ChainIDs.Chiado]
+  ValidatorDashboardNetworkList = [ChainIDs.Chiado, ChainIDs.Holesky]
```
Comment (D13ce) [inline frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue:15]: **Question:** What is the reason for this change?

File: frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue [hunk 2]
```diff
@@ -20,19 +22,19 @@ const selection = ref<string>('')
...
 
 const buttonList = ValidatorDashboardNetworkList.map((chainId) => {
+  // TODO: simply set `false` for everything once dashboards can be created for all the networks in `ValidatorDashboardNetworkList`
+  const isDisabled = !useRuntimeConfig().public.showInDevelopment && chainId !== currentNetwork.value
   return {
     value: String(chainId),
     text: ChainInfo[chainId].family as string,
-    subText: (ChainInfo[chainId].name !== ChainInfo[chainId].family as string) ? ChainInfo[chainId].name : ChainInfo[chainId].description,
-    disabled: !useRuntimeConfig().public.showInDevelopment && chainId !== currentNetwork.value, // TODO: simply set `false` for everything once dashboards can be created for all the networks in `ValidatorDashboardNetworkList`
+    subText: isDisabled ? $t('dashboard.creation.coming_soon') : ChainInfo[chainId].name,
```
Comment (D13ce) [inline frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue:30]: **Bug:** Using `isDisabled` like this is good but instead of `: ChainInfo[chainId].name` you will have to use everything that was used for `subText` before, namely `(ChainInfo[chainId].name !== ChainInfo[chainId].family as string) ? ChainInfo[chainId].name : ChainInfo[chainId].description`.

Because without that, the Ethereum button looks like this
image
while it should look like this
image

File: frontend/i18n/en.json [hunk 3]
```diff
@@ -366,7 +366,8 @@
       },
       "network": {
         "subtitle": "What network are your validators on?"
-      }
+      },
+      "coming_soon": "Coming soon"
```
Comment (D13ce) [inline frontend/i18n/en.json:370]: **Request:** There is no need for this, we already have https://github.com/gobitfly/beaconchain/blob/0cc4ea4e81955f4b725c33e0b3cef2d1cec3ad9b/frontend/i18n/en.json#L46
Please delete the new addition and make use of the one in `common`.

**Review by D13ce — CHANGES_REQUESTED** [review]

File: frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue (hunk 1 above)
Comment (marcel-bitfly) [reply frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue]: as noted in the ticket: active Buttons should appear left

File: frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue (hunk 2 above)
Comment (marcel-bitfly) [reply frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue]: thx done

File: frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue [hunk 4]
```diff
@@ -3,36 +3,47 @@ import { IconNetwork } from '#components'
...
+  selection.value = `${ChainIDs.Ethereum}` // preselecting here, because Ethereum (Mainnet) is the only network that is available at the moment
 } else {
   ValidatorDashboardNetworkList = [ChainIDs.Holesky, ChainIDs.Chiado]
+  selection.value = `${ChainIDs.Holesky}` // preselecting here, because Ethereum (Holesky) is the only network that is available at the moment
 }
 // ... and remove this.
 
-const network = defineModel<ChainIDs>('network')
-const selection = ref<string>('')
+watch(selection, (value) => { network.value = Number(value) as ChainIDs }, { immediate: true })
```
Comment (marcel-bitfly) [inline frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue:23]: fyi: `immediate` is necessary as we are preselecting now

**Review by D13ce — APPROVED** [review]
LGTM

+1 for the UX improvement along the way :+1:
